dss/events/handlers/sync.py

```diff
@@ -1,20 +1,35 @@
+import os
+import sys
 import datetime
+import json
 import time
 import hashlib
 from contextlib import closing
+from concurrent.futures import ThreadPoolExecutor
+from collections import namedtuple
 
 import boto3
 import botocore.session
 import urllib3
 import google.cloud.storage
+from google.resumable_media._upload import get_content_range
 from google.cloud.client import ClientWithProject
 from google.cloud._http import JSONConnection
 
 from dss import Config
+from dss.util.aws import resources, clients, send_sns_msg, ARN
+from dss.util.streaming import get_pool_manager, S3SigningChunker
+from dss.blobstore.gs import GSBlobStore
 
 presigned_url_lifetime_seconds = 3600
 use_gsts = False
```

Comment:
It depends on an economics evaluation of the system. It's possible, but unlikely, that we'll end up in a regime where GSTS is substantially cheaper than pushing data around with Lambdas, and doesn't impact QoS much, but to do that we'd probably need payload batching of some sort.

I've filed #274 to remove this code in a separate PR and save it in a branch for future reference.